src/libs/copy_history.c

```diff
@@ -368,7 +368,7 @@ void gui_init(dt_lib_module_t *self)
   gtk_grid_set_column_homogeneous(grid, TRUE);
   int line = 0;
 
-  d->copy_parts_button = dt_ui_button_new(_("copy parts..."),
+  d->copy_parts_button = dt_ui_button_new(_("selective copy..."),
                                           _("copy part history stack of\nfirst selected image"),
```

Comment:
I suggest to update tooltips too and remove part word to replace it. Something like: "select modules to copy" (instead of copy part histor stack of). @elstoc: as native english speaker, if you have a better idea for tooltips...

Reply:
choose which modules to copy from the source image / choose which modules to paste to the target image(s)

Reply:
@elstoc : Thanks, I'll commit that.